core/domain/config_domain.py

```diff
@@ -274,3 +274,7 @@ def get_config_property_schemas(cls):
 RECORD_PLAYTHROUGH_PROBABILITY = ConfigProperty(
     'record_playthrough_probability', FLOAT_SCHEMA,
     'The probability of recording playthroughs', 0.2)
+
+IS_ISSUES_TAB_EXPOSED = ConfigProperty(
```

Comment:
Rather than 'EXPOSED' could we maybe use 'ENABLED'? That's a bit more conventional for flags used to gate functionality.